.github/workflows/CI.yml

```diff
@@ -27,30 +27,62 @@ jobs:
         os:
           - ubuntu
           # Windows users, feel free to open a PR :)
+          # See issue #242, windows support would be welcome.
           # - windows
           - macos
-        exclude:
-          - os: windows # See issue #242, windows support would be welcome.
+        geos:
+          - '3.12.0'
+          - '3.11.2'
+          - '3.10.5'
+          - '3.9.4'
+          - '3.8.3'
     runs-on: ${{ matrix.os }}-latest
     continue-on-error: ${{ matrix.ruby == 'head' || matrix.os == 'windows' || matrix.os == 'macos' }}
-    name: Ruby ${{ matrix.ruby }} (${{ matrix.os }})
+    name: Ruby ${{ matrix.ruby }}, Geos ${{ matrix.geos }} (${{ matrix.os }})
     steps:
       - uses: actions/checkout@v3
-      - uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: ${{ matrix.ruby }}
-          bundler-cache: true
-      - name: Install Geos (Linux)
+      - name: Install Packages (Linux)
         if: matrix.os == 'ubuntu'
         run: |
           sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
-          sudo apt-get install -yqq libgeos-dev
-      - name: Install Geos (Mac)
+          sudo apt-get install curl libcurl4-openssl-dev libssl-dev cmake
```

Comment:
I'd keep the `-yqq` because this avoids having logs too large, which we don't need anyway :)

Reply:
done